### PR TITLE
feat(filter): filter movies by title

### PR DESCRIPTION
**Pull request by nathankhuu**
What: filter by exact or fuzzy title
Why: last filter for get /movies
Details:
Created migration to add fuzzystrmatch extension
Validated title query and use title to filter
Notes:
Need to be superuser to add extensions
https://lobsters.atlassian.net/browse/ENG-7047

### CODE REVIEW

**Comment by lvamaral** [comment]
is this part of miyagi?

**Comment by nathankhuu** [comment]
yeah it's a part of miyagi

I don't think filtering by fuzzy title was listed in the miyagi spec, but it's mentioned on jira

**Comment by lvamaral** [comment]
@nathankhuu could you post the jira ticket link on the description?

**Comment by nathankhuu** [comment]
@lvamaral just updated it!

**Review by lvamaral — APPROVED** [review]

**Comment by lvamaral** [comment]
lgtm!

File: lib/plugins/features/movies/controller.js [hunk 1]
```diff
@@ -17,6 +18,9 @@ exports.list = (query) => {
     if (query.end_year) {
       movieQuery.where('release_year', '<=', query.end_year);
     }
+    if (query.title) {
+      movieQuery.where(Knex.raw(`levenshtein(name, '${query.title}') <= 1`));
```
Comment (jalcine) [inline lib/plugins/features/movies/controller.js:22]: Nice!

**Review by jalcine — APPROVED** [review]
LGTM

**Pull request merged**